### PR TITLE
Fixed Null Pointer Exception

### PR DESCRIPTION
**Pull request by inaseem**
I Was Getting A Null Pointer Exception On Compiling And Running A File That I Have Just Created Using Notiler. I Fixed The Exception. Do Review The Changes.

### CODE REVIEW

**Comment by inaseem** [comment]
Removed Unused Imports And Re Indented The Code Also.

File: src/Notepad.java [hunk 1]
```diff
@@ -1319,38 +1310,37 @@ public void compileFile() {
...
+                    b = true;
                 }
                 din.close();
-                if(!b){
+                if (!b) {
                     jTextArea2.setText("No Syntax Error");
                 }
             } catch (Exception e) {
             }
-        }
```
Comment (Tanmay17) [inline src/Notepad.java]: This is a C+Java compiler

Comment (naseemali925) [reply src/Notepad.java]: Yes I Know. But Whenever You Save The Files. Your dir and name Variables Were Not Changing.

Comment (naseemali925) [reply src/Notepad.java]: And Thats Why It Was Giving Null Pointer Exception Whenever You Compile And Run Any File That Has Been Created And Saved With Notiler  Menu.

File: src/Notepad.java [hunk 2]
```diff
@@ -1545,133 +1535,126 @@ private void jTextArea1MousePressed(java.awt.event.MouseEvent evt) {//GEN-FIRST:
...
+                    int len1 = txt.length();
+                    byte buf[] = txt.getBytes();
+                    File f1 = new File(path);
+                    FileOutputStream obj = new FileOutputStream(f1);
+                    for (int i = 0; i < len1; i++) {
+                        obj.write(buf[i]);
+                    }
+                    obj.close();
+                    this.setTitle(file);
+                    name = file;
```
Comment (Tanmay17) [inline src/Notepad.java:1561]: Instead of "name" initialize "nam = file"

**Comment by inaseem** [comment]
No Sir. You Need To Keep That name=file As It Is.
I Have Already Tested With The Change You Suggested. But Nothing Seems To Work.
Except The Change I Have Created. And Thats Why I Created A PR.

**Review by Tanmay17 — APPROVED** [review]

**Pull request closed without merging**